garden-backend-service/src/api/schemas/modal_app.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+from uuid import UUID
+
+from pydantic import AliasPath, Field, computed_field
+
+from .base import BaseSchema, UniqueList
+from .modal_function import ModalFunctionMetadataResponse, ModalFunctionMetadata
+
+class ModalAppMetadata(BaseSchema):
+    app_name: str
+    version: str = "0.0.1"
+    modal_function_names: list[str] = Field(default_factory=list)
+    file_contents: str
+
+    is_archived: bool = False
+
+    requirements: list[str] = Field(default_factory=list)
+    base_image_name: str
```

Comment:
understanding check: where do we get this field from? is it part of the request payload, or would we extract it from the deployed app?